zuul.d/nodesets.yaml

```diff
@@ -582,8 +582,8 @@
 - nodeset:
     name: openvswitch-2.9.0-python38
     nodes:
-      - name: ubuntu-bionic
-        label: ubuntu-bionic-1vcpu
+      - name: centos-8-stream
```

Comment:
lets delete all the other openvswitch nodesets. They are no longer needed

Reply:
Ack.